client/src/utils/colorBrightness.ts

```diff
@@ -0,0 +1,43 @@
+import {DEFAULT_THEME} from '../components/ThemeDropdown'
+
+/**
+ * Converts the @see hexColor from HEX to RGB.
+ * @param hexColor The HEX input color
+ * @returns The RGB color
+ */
+function hexToRgb(hexColor: string) {
+    var result = /^#?([a-f\d]{2})([a-f\d]{2})([a-f\d]{2})$/i.exec(hexColor);
+    var r = result ? parseInt(result[1], 16) : 0;

```

Comment:
Please always use `let` instead of `var` (var is available just for legacy support)

Just a small reformatting to avoid having unused variables
```ts
/**
 * Converts the @see hexColor from HEX to RGB.
 * @param hexColor The HEX input color
 * @returns The RGB color
 */
function hexToRgb(hexColor: string) {
  const result = /^#?([a-f\d]{2})([a-f\d]{2})([a-f\d]{2})$/i.exec(hexColor);

  if (result?.length !== 4) return null;

  return {
    r: parseInt(result[1], 16),
    g: parseInt(result[2], 16),
    b: parseInt(result[3], 16),
  };
}

/**
 * The brightness treshold
 */
export const BRIGHTNESS_TRESHOLD = 100;

/**
 * Computes the pseudo brightness from the @see hexColor. For more see the @link https://stackoverflow.com/a/11868159
 * @param hexColor The HEX input color
 * @returns The brightness
 */
export function getHexBrightness(hexColor: string) {
  const rgb = hexToRgb(hexColor);
  return rgb ? Math.round((rgb.r * 299 + rgb.g * 587 + rgb.b * 114) / 1000) : 0;
}
```

Reply:
Fixed